UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/appsettings.json

```
@@ -84,5 +84,6 @@
     "SleepDuration": 2
   },
   "AllowedHosts": "*",
-  "APPINSIGHTS_INSTRUMENTATIONKEY": ""
+  "APPINSIGHTS_INSTRUMENTATIONKEY": "",
+  "ValidPastWeeks": 4
```

This configuration item name isn't very clear and it's meaning will be lost/confusing.

Can I suggest:


```suggestion
  "MaximumNumerOfDaysValidForSinceDateTimeEndpoint": 28
```

---

Resolved.